frontend/packages/kubevirt-plugin/src/components/vms/utils.tsx

```diff
@@ -0,0 +1,3 @@
+import * as React from 'react';
+
+export const Dashes = () => <span className="text-muted">-</span>;
```

Comment:
why is this necessary? Isn't just a string enough?

- nevertheless this should be exported as a component from `@console/shared`

Reply:
Sorry I might have been wrong on using `text-muted` here. We should do whatever is done elsewhere. (Hopefully we're consistent.)

Reply:
ok, changed to use single dash now.